### ISSUE TITLE
Incompatible with Rollup

### ISSUE THREAD

**Issue by e-e-e** [rdfjs/N3.js#134]
When attempting to bundle N3 into another app using Rollup, it throws this error:
```
[!] Error: Illegal reassignment to import 'commonjsHelpers'
node_modules/sparql-iterator/node_modules/n3/N3.js (3:0)
1: // Replace local require by a lazy loader
2: var globalRequire = require;
3: require = function () {};
   ^
4: 
5: // Expose submodules
``` 
It looks like the method employed for lazy loading causes issues with their commonjs resolver.
I am unsure of another alternative at the moment. Perhaps exposing an non-lazy version of N3 would be a hacky start towards solving this issue.

**Comment by RubenVerborgh** [comment]
Mmm, maybe I should just drop the lazy loading in general. I don't think it's necessary anymore.

**Comment by e-e-e** [comment]
That would be excellent. Its kind of dangerous territory now to mess around with import/require.

**Comment by RubenVerborgh** [comment]
Agree, there's no way I can keep track of all build systems these days. Worked with webpack though (but could probably break in any future release).

**Comment by RubenVerborgh** [comment]
Released as 1.0.0-beta.1
